### ISSUE TITLE
Personal activity log

### ISSUE THREAD

**Issue by ktlinux** [friendica/friendica#12610]
### Is the feature request related to a problem? Please describe.

Well, the current way of seeing your previous activity on Friendica feels rather cumbersome, as you need to go to the "Personal" tab and scroll all the way through, to find a comment you posted. Sometimes you just want to check on a post and find a comment of yours for reference and if you have a lot of activity, this is quite not possible.

### Describe the feature you'd like

I'd like to have an option of seeing an "Activity log" similar to Facebook's (see screenshot below, personal accounts, including mine, edited for privacy reasons) where you can see your entire interaction history on Friendica, at least to a certain time - this includes likes, dislikes, comments, reshares, quote shares, posts of you, etc. and with a link to get to these posts. The content should not be displayed entirely, rather just the first 2 lines of it or something like that. This would also make Friendica more easily recommendable for people on Facebook.

Please let me know what you think!

![activity_log](https://user-images.githubusercontent.com/27150141/210509145-f675e953-ec96-4895-baea-2af0d18b3806.png)

**Comment by tobiasd** [comment]
Do you mean something like `/notifications/system?show=all` and the other tabs of the notification page? Just expanded by the 1st two sentences of the postings listed there?

**Comment by ktlinux** [comment]
> Do you mean something like `/notifications/system?show=all` and the other tabs of the notification page? Just expanded by the 1st two sentences of the postings listed there?

Oh wow, did not know this trick, haha, but indeed, it would be really helpful to have a little preview on the content that has been interacted with, as this would be easier for locating everything.

**Comment by SpcCw** [comment]
I think the limitation of existing system notifications feed is that it shows items you interacted/created _if and after_ something else happened to them and sorts them in that order.

E.g. if you liked/commented some post and then nothing else happened in the thread - you won't see it listed there.

The feature OP describes is more like history log of your own activity so you can backtrace your actions in historical order, e.g. "I saw that video this Saturday and liked it, now I want to find it again!" - notification feed probably won't help here. Another tricky thing are media posts (when there is picture/video/sound and not much else) because short text abstracts in notifications make it hard to identify them even when you see them.

Another peculiar thing: apparently you can see _posts and comments_ feed for any contact but not yourself so it is hard to find something you commented unless there are notifications or you can use search.

**Comment by MrPetovan** [comment]
Tobias was only suggesting how the personal activity log should look like, he didn't imply the system notification feature was covering the original request.

**Comment by ktlinux** [comment]
> The feature OP describes is more like history log of your own activity so you can backtrace your actions in historical order, e.g. "I saw that video this Saturday and liked it, now I want to find it again!" - notification feed probably won't help here.

That is correct. I see however that the whole section sits under the "personal" tab, then "show all", so this kinda is what I am looking for. It is not really intuitive, however, and there is no way I find how to access them rather than pasting the URL end in the browser's URL bar, which is harder on mobile. It would be great to have a button or a hyperlink in the notification pane (like "show all notifications" or something) that would take you to `/notifications/system?show=all`, and from there let the users go wherever they want - that is a solution I see, if there is no UI to get there already.

Also, it would be great to update the help section if possible, with this information, whether there is a UI to get there or not. 

> Another tricky thing are media posts (when there is picture/video/sound and not much else) because short text abstracts in notifications make it hard to identify them even when you see them.

Yea, this is a problem as well, but I can see this fixable by replacing the profile pictures of the users involved with thumbnails (for pictures), thumbnails with a play button overlay (for videos) or a music icon logo (for music). If multiple media of the same type are in the post, the thumbnail would be just the first one (e.g. 1st photo in the post), I don't know (maybe this should fit in a different issue, though, indeed).

Another issue that I see quite related to this, however, that I cannot see an easy fix for, is that I cannot see any way in the UI to filter between my comments, my likes, my dislikes etc. They are all in the same stream of notifications.
